components/collection/drop/PaidGenerative.vue

```diff
@@ -106,17 +106,8 @@ const handleSubmitMint = async () => {
     return false
   }
 
-  openMintModal()
-  await massGenerate()
-}
-
-const openMintModal = () => {
   isMintModalActive.value = true
-}
-
-const handleMintModalClose = () => {
-  closeMintModal()
-  clearMassMint()
```

Comment:
now we are not clearing the massmint on modal close,

Reply:
updated